openassessment/xblock/xml.py

```diff
@@ -9,9 +9,9 @@
 
 import dateutil.parser
 import defusedxml.ElementTree as safe_etree
+from lxml import etree
```

Comment:
Pylint complaining about unnecessary `as` import statement.

Reply:
so, is this used at all? should it just be removed completely?

Reply:
It is used. See line 80.